### ISSUE TITLE
Installer failing with hardware error

### ISSUE THREAD

**Issue by fabricio7p** [elementary/installer#466]
Installer fails with "Could not install" 
Message: "Installing elementary OS 6 early access failed, possibly due to a hardware error..."

PC:
XPS 13 9300, i7 1065G7, 1gb SSD

Partition setup:
/dev/nvme0n1p1: 16gb SWAP
/dev/nvme0n1p2: 1gb FAT32 efi (for arch)
/dev/nvme0n1p3: 300gb ext4 (arch)
/dev/nvme0n1p4 1gb FAT32 efi (for gentoo)
/dev/nvme0n1p5 300gb ext4 (gentoo)
**/dev/nvme0n1p6 1gb fat32 efi (for elementary)**
**/dev/nvme0n1p7 336gb ext4 (elementary)**

Details info:

```
INFO: running "chroot" "/tmp/distinst.mfSxvTDrmawq" "apt-get" "install" "-q" "-y" "-o" "Acquire::cdrom::AutoDetect=0" "-o" "Acquire::cdrom::mount=/cdrom" -o" "APT:CDROM::NoMount=1 "grub-efi" "grub-efi-amd64" "grub-efi-amd64-signed" "shim-signed" "mokutil" "fwupdate-signed" "linux-signed-generic" "e2fsprogs" "dosfstools"
ERROR: configuring chroot error: error installing packages: command failed with exit status: exit code: 100
```

- grab a completely fine pendrive,
- flash any build with balena-etcher on arch linux.
- attempt to install on the end of your storage drive

obs: i typed the details info by hand because i couldn't copy paste from the installer

**Comment by mrgian** [comment]
Same error even with an unpartitioned disk (unallocated space only)

**Comment by Slaymo** [comment]
Yep - same here, happens to me (on two separate systems) on all previous 6.0 early release builds for me, from the 16th of November onwards. Worked fine until then!!

**Comment by ryonakano** [comment]
I did also face this issue but successfully installed with the old iso published at November 27: I partitioned my ssd into the efi, root, and home partition and installation succeeded.

**Comment by fabricio7p** [comment]
> Yep - same here, happens to me (on two separate systems) on all previous 6.0 early release builds for me, from the 16th of November onwards. Worked fine until then!! One thing I'll note is that the extraction process goes WAY too quickly, at least in my experience with other squashfs installs. Which may point to a broader problem than simply installing packages to a boot partition. It flies by, which would normally be a good thing - but it is suspiciously quick.

Yeah i did noticed that too. I changed my partition order to be first the root "/" partition and then efi "/boot". The file copying process seemed waay slower, more like it should be actually. But then got the same error at the end

**Comment by axiel7** [comment]
Same problem here, I've even tried with my own compiled build

**Comment by Slaymo** [comment]
> I did also face this issue but successfully installed with the old iso published at November 27: I partitioned my ssd into the efi, root, and home partition and installation succeeded.

Ok - trying this method now, gimme about 20-30 minutes and I'll let you know how I go!

**Comment by axiel7** [comment]
> I did also face this issue but successfully installed with the old iso published at November 27: I partitioned my ssd into the efi, root, and home partition and installation succeeded.
> 

I can confirm this method works even with only efi and root partitions

**Comment by fabricio7p** [comment]
> > I did also face this issue but successfully installed with the old iso published at November 27: I partitioned my ssd into the efi, root, and home partition and installation succeeded.
> 
> I can confirm this method works even with only efi and root partitions

You manage to install newer releases or only the nov 27?
Anyways, i flashing nov 27 here to try aswell

**Comment by Slaymo** [comment]
Ok - so I can confirm the the 27th Nov installer does work for this issue. It was a little ugly, but it worked. Here's what I did:

1. Burnt to iso using gnome disk utility
2. Booted into eos 6.0 installer, took a while (quite a while) for first install dialog to appear, was just grey with notifications up top. But it appeared after a bit.
3. Partitioned WITHIN the installer, not using pre-made partition scheme (that may be key). When gparted loaded up it initially had 'caution' symbols next to all partitions on all disks. Making a change seemed to rectify that.
4. Install went past the initial problem of installing boot packages, but then ended with an ostensible 'cannot install bootloader' issue. I say 'ostensible' because rebooting and going into my efi boot selector revealed the newly installed 6.0 odin release. Booted up fine, all seems ok.

**Comment by fabricio7p** [comment]
> Ok - so I can confirm the the 27th Nov installer does work for this issue. It was a little ugly, but it worked. Here's what I did:
> 
> 1. Burnt to iso using gnome disk utility
> 2. Booted into eos 6.0 installer, took a while (quite a while) for first install dialog to appear, was just grey with notifications up top. But it appeared after a bit.
> 3. Partitioned WITHIN the installer, not using pre-made partition scheme (that may be key). When gparted loaded up it initially had 'caution' symbols next to all partitions on all disks, the caution was 'cannot read partition' in information section. Making a change seemed to rectify that.
> 4. Install went past the initial problem of installing boot packages, but then ended with an ostensible 'cannot install bootloader' issue. I say 'ostensible' because rebooting and going into my efi boot menu/selector revealed the newly installed 6.0 odin release. Booted up fine, all seems ok.

Yep, we had the exact same experience. nov 27 build does work fine here aswell

**Comment by pmk1c** [comment]
I'm having the same issue. Since Nov 27 builds are now unavailable, I'm completely unable to install. Dec 03 image seems to have the same problem.

I saved the log files that were created in the "writable" partition here, if someone is interested, I can send them.

**Comment by xeu100** [comment]
Same issue, tried building from older source but to no avail.

**Comment by mrgian** [comment]
I built the iso from source today and it's now correctly installing
However I had to manually install io.elementary.terminal package

**Comment by xeu100** [comment]
I made an issue on the appcenter repository about pantheon-terminal not working properly, due to some dependency resolution stuff.

**Comment by xeu100** [comment]
Update: Installer does not work and leaves the system in an unbootable state. Gets caught in grub, will try to manually recover through the USB.

**Comment by ryonakano** [comment]
I recently tried a fresh installation with the latest iso to the machine that I'd tried and failed before and it works. Is anyone here still facing the issue?
